src/applications/mhv-supply-reordering/actions/index.js

```diff
@@ -27,19 +41,19 @@ export const fetchMdotData = () => async dispatch => {
   dispatch(initiateApiCall());
   apiRequest(MDOT_API_URL)
     .then(body => {
-      if (body.errors) {
-        const mdotError = head(body.errors);
+      if (body.errors && body.errors.length > 0) {
```

Comment:
I was thinking object/array destructuring, e.g. `let { errors : [mdotError = null] = [] } = body;` or similar, but what you have is more readable.